api/envoy/data/core/v2alpha/BUILD

```diff
@@ -13,3 +13,16 @@ api_proto_library(
         "//envoy/api/v2/core:base",
     ],
 )
+
+api_proto_library(
```

Comment:
I'm not sure if I would consider this a core type. Is this a generic JSON description or very specific to the idea of returning a JSON reply from HCM?